examples/substrate-react-example/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
this is to avoid the re-write on the config?

Reply:
this is totally okay to have, even is recommended

@wainola please read these 2 documents below
https://developer.mozilla.org/en-US/docs/Glossary/Doctype
- https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode

Reply:
this is totally okay to have, even is recommended

@wainola please read these 2 documents below
https://developer.mozilla.org/en-US/docs/Glossary/Doctype
- https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode